common/src/main/resources/config/thunibib-common/messages_de.properties

```diff
@@ -64,6 +64,7 @@ ubo.edition                                     = Ausgabe
 ubo.edition.out                                 = Ausgabe
 ubo.edition.placeholder                         = z. B. "3. Auflage" oder Bezeichnung der Ausgabe
 ubo.genre                                       = Dokumenttyp
+ubo.help.accessrights                           = Geben Sie hier bitte die Zugangsrechte (Access Rights) der Publikation an. Eine genauere Erkl\u00E4rung der Zugangsrechte finden Sie im Abschnitt FAQ der Seite <a href="faq.xml">\u00DCber die Bibliographie</a>.
```

Comment:
In der FAQ muss dann noch er entsprechende Text ergänzt werden. Der Kölner Text kann hier vermutlich als Vorlage genutzt werden:
https://github.com/gbv/ubo_koeln/blob/a132a46f5bab93c8ba84634b645103e779430a08/src/main/resources/META-INF/resources/about.xml#L274-L301

Reply:
@kkrebs waren in common schon drin, habe sie jetzt aber für J, EF und IL hinzugefügt